nekoyume/Assets/_Scripts/UI/Widget/Hud/SpeechBubble.cs

```diff
@@ -40,6 +40,7 @@ protected override void Awake()
 
         protected override void OnDisable()
         {
+            StopAllCoroutines();
```

Comment:
I think this line is not necessary because all of the coroutines depends of this `MonoBehaviour` will be stop when `OnDisable()` invoked. Is there any issues?